frontend/src/metabase/css/query_builder.module.css

```diff
@@ -69,174 +52,169 @@
   padding: 0;
 }
 
-.TooltipFilterList .Query-filter {
+:global(.TooltipFilterList .Query-filter) {
   padding-left: 0 !important;
   padding-bottom: 0 !important;
 }
 
-.TooltipFilterList .Query-filterList {
+:global(.TooltipFilterList .Query-filterList) {
   flex-direction: column;
 }
 
-.TooltipFilterList .Query-section {
```

Comment:
I still found these 2 classes in use. Did you remove this rule or move it somewhere else?

https://github.com/metabase/metabase/blob/a1df251cd75d454e4f56d0b14993e59bd943af14/frontend/src/metabase/admin/datamodel/components/QueryDefinitionTooltip/QueryDefinitionTooltip.jsx#L30

https://github.com/metabase/metabase/blob/a1df251cd75d454e4f56d0b14993e59bd943af14/frontend/src/metabase/admin/datamodel/components/GuiQueryEditor/GuiQueryEditor.jsx#L129

https://github.com/metabase/metabase/blob/a1df251cd75d454e4f56d0b14993e59bd943af14/frontend/src/metabase/admin/datamodel/components/GuiQueryEditor/GuiQueryEditor.jsx#L215